websites/D/Dropout/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "189690228292845568",
+		"name": "angelolz"
+	},
+	"service": "Dropout",
+	"description": {
+		"en": "Independent, ad-free, uncensored original comedies that you won't find anywhere else."
+	},
+	"url": "dropout.tv",
+	"matches": [
+		"*://dropout.tv/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.ibb.co/hCK6Lj3/dropout-logo.png",
```

Comment:
Website is unavailable?
![image](https://github.com/PreMiD/Presences/assets/42322979/2a03f713-a86b-478d-b34e-e57db17c9048)

Reply:
Hmm, not sure why that is, works fine for me. I'll try another image hosting site.

Reply:
Updated site.